schemas/content/product-attribute.schema.json

```diff
@@ -0,0 +1,52 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/content/product-attribute",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Product Attribute",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Generic product attribute of a product in the product catalog.",
+  "definitions": {
+    "attribute": {
+      "properties": {
+        "xdm:key": {
+          "title": "Key",
+          "type": "string",
+          "description": "The technical key / identifier of this attribute."
+        },
+        "schema:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "The human-readable name of the attribute."
+        },
+        "xdm:value": {
```

Comment:
This has no type. That is not allowed.

Reply:
Can we define "object" as type. Product variants can be strings, numbers, dates, etc. thats why I left this out.

Reply:
Since "object" is no valid type. How can we make this generic? Since defining this as string for example is to short.

Reply:
I added `"type": "string"` for now, but I'm not really happy with it. I think there should be a type=object or type being optional, to have the flexibility to have product attributes with different types like number, date, etc.

Reply:
You can `oneOf` multiple `type`s.

Reply:
Changed to 'oneOf`.